src/handlers/access/labels-access.ts

```diff
@@ -33,12 +33,12 @@ export const handleLabelsAccess = async () => {
 
     if (payload.action === "labeled") {
       // remove the label
-      removeLabel(payload.label?.name!);
+      await removeLabel(payload.label?.name!);
```

Comment:
Maybe we should ban using non-null assertions via eslint because I see them being abused quite a bit in this codebase.